uiowa/uiowa
# Feature: Styling area of study admissions requirements

Resolves https://github.com/uiowa/uiowa/issues/2934. 

# How to test

- `blt ds --site=admissions.uiowa.edu`
- Go to http://admissions.local.drupal.uiowa.edu/areas-of-study/accounting and add admission requirements and verify that they are styled like the screenshot below:

<img width="1683" alt="Screen Shot 2021-02-04 at 4 29 57 PM" src="https://user-images.githubusercontent.com/1036433/106963754-ade71780-6706-11eb-9fae-c7efbdf4f36a.png">
